src/Workspaces/CoreTest/SyntaxReferenceTests.cs

```diff
@@ -26,10 +25,7 @@ private static Workspace CreateWorkspace(Type[] additionalParts = null)
             => new AdhocWorkspace(FeaturesTestCompositions.Features.AddParts(additionalParts).GetHostServices());
 
         private static Workspace CreateWorkspaceWithRecoverableSyntaxTrees()
-            => CreateWorkspace(
-            [
-                typeof(TestTemporaryStorageServiceFactory)
-            ]);
+            => CreateWorkspace();
```

Comment:
same.  thsi helper is now pointless.

Reply:
Why even have this method and not just have the callers call CreateWorkspace themselves?

Reply:
Yup yup. That's the goal. These helpers used to do things. But over time those capabilities went away or changed. Plus, we want our tests testing real code as much as possible. 

I'm just going to do it in separate pr :)